### ISSUE TITLE
Improve docs on how to enable developer mode

### ISSUE THREAD

**Issue by MorganRodgers** [OSC/ood-documentation#156]
From this thread: https://discourse.osc.edu/t/enabling-app-development/322/1

- Itemize the three steps needed to  enable the app development as subsections, and detail what needs to be  done and what the options are under each subsection (e.g. for the app  paths, either the 1.3 or 1.4 way).
- especially for the paths, make another  sub-itemization separately for the 1.3 and 1.4 behavior, so one does  not confuse the 1.3 behavior with the 1.4.
- Then have a separate subsection wrapped around the description of Table 7 = the global enablement of the apps development
- it is not clear if one needs to set  OOD_DEV_APPS_ROOT (and where this variable goes - presumably in the  /etc/ood/config/apps/dashboard/env) AND also modify the nginx_stage.yml  appropriately). This needs some clarification too.
- it is not clear OOD_DEV_SSH_HOST needs  to be set if I set up the apps on the same OOD server itself. It sounds  like in that case OOD_DEV_SSH_HOST is not needed, but, then it should  be listed as an “optional” step.

https://osc.github.io/ood-documentation/master/app-development/enabling-development-mode.html

**Comment by diedpigs** [comment]
In the section of restricting app development mode to group membership, the code should be:

```
Configuration.app_development_enabled = OodSupport::Process.groups.include?(
  OodSupport::Group.new("devgrp")
)
```

**Comment by MorganRodgers** [comment]
@diedpigs fixed in 1159749. Thanks for pointing that out.

**Comment by MorganRodgers** [comment]
Be sure to include the changes from: https://github.com/OSC/ood-dashboard/pull/478
